### ISSUE TITLE
Leaking Response Body

### ISSUE THREAD

**Issue by Jawnnypoo** [aurae/retrofit-logansquare#8]
Using `compile "com.github.aurae.retrofit2:converter-logansquare:1.2.0"`  I am getting an error in Logcat:

```
A connection to https://my-url.com/blah was leaked. Did you forget to close a response body?
```

I believe this has to do with `LoganSquareResponseBodyConverter.convert` not closing its InputStream. I will investigate this further, just wanted to make you aware for now.

**Comment by aurae** [comment]
Thank you for your report! Indeed, the call to `close()` is missing on the `ResponseBody` being passed in to the response converter, and that would be the primary culprit for this warning.

I will be wrapping the entirety of `LoganSquareResponseBodyConverter#convert` in a `try/finally` block, with the call to `close()` in the finally block to ensure execution.

**Comment by Jawnnypoo** [comment]
Awesome, thanks for the quick turnaround!

**Comment by aurae** [comment]
This has been released as 1.2.1.
